pkg/sync/sync.go

```diff
@@ -171,101 +173,173 @@ func ListAll(store object.ObjectStorage, start, end string) (<-chan object.Objec
 
 var bufPool = sync.Pool{
 	New: func() interface{} {
-		buf := make([]byte, 32<<10)
+		buf := make([]byte, defaultPartSize)
```

Comment:
@davies not sure if this change is reasonable

Reply:
Don't, defaultPartSize is too large as an buffer